.github/workflows/sycl-linux-precommit.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       build_artifact_suffix: "default"
       build_cache_suffix: "default"
       # Docker image has last nightly pre-installed and added to the PATH
-      build_image: "ghcr.io/intel/llvm/sycl_ubuntu2204_nightly:build"
+      build_image: "ghcr.io/intel/llvm/sycl_ubuntu2404_nightly:latest"
```

Comment:
can we just delete the `build` image?

Reply:
discussed offline, we only create `*nightly:latest` now, the `:build` is getting increasingly stale/outdated.